pkg/handlers/s3_dev_handler.go

```diff
@@ -179,7 +181,7 @@ func (h *s3DevHandler) OnAdd(ns string, vc kvcv1.VolumeConfig, controllerRef met
 	}
 }
 
-func (h *s3DevHandler) OnDelete(ns string, vc kvcv1.VolumeConfig, controllerRef metav1.OwnerReference) {
+func (h *s3DevHandler) OnDelete(ns string, vc kvcv1.VolumeConfig, vStatus kvcv1.Volume, controllerRef metav1.OwnerReference) {
```

Comment:
We have to spin up the deletion pod here too right?

Reply:
During offline discussions, we agreed to remove s3-dev type. Ignoring this comment.